modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsFullMessage.java

```diff
@@ -361,7 +355,7 @@ public IgniteDhtPartitionHistorySuppliersMap partitionHistorySuppliers() {
     /**
      *
      */
-    public Set<Integer> partsToReload(UUID nodeId, int grpId) {
+    public Collection<Integer> partsToReload(UUID nodeId, int grpId) {
         if (partsToReload == null)
```

Comment:
Let's replace method body by `F#emptyIfNull` call. Also it is actual for `#partitionSizes` method (if you add `F#emptyIfNull` for maps).